src/proc/typifier.rs

```diff
@@ -298,6 +298,7 @@ impl<'a> ResolveContext<'a> {
                             width,
                             space,
                         },
+                        Ti::BindingArray { base, .. } => Ti::Pointer { base, space },
```

Comment:
There should never be a `Pointer` to a `BindingArray`, right? Why isn't the non-pointer `BindingArray` case that's already here good enough?

(Similar question about the similar change below.)

Reply:
For all globals in non-`Handle` address space the resolved type is a pointer. See the typifier code.